README.md

```diff
@@ -43,10 +43,12 @@ $ pip install -r requirements.txt
 When uritemplate is not installed, a local implementation is used instead.
 But the local stub supports only the most basic [URI Templates][rfc6570].
 
-## Using the Python 3 ref-engine discovery tool
+## Using ref-engine discovery tool
```

Comment:
This is right after the Pythpn-specific dependencies section.  I think it's useful to have an example resolution in the top level, but we may want to pull the bulk of the Python docs into `oci_discovery/README.md` to mirror `tools/README.md`.